app/components/CommitNavbar.scss

```diff
@@ -15,21 +15,10 @@
   background-position-x: 60px;
 }
 
-.clear {
```

Comment:
If we want to take off the X sign then we can delete it from assets as well

Reply:
it's not in assets